bindsnet/analysis/plotting.py

```diff
@@ -410,7 +410,7 @@ def plot_voltages(voltages: Dict[str, torch.Tensor], ims: Optional[List[AxesImag
                 if plot_type == 'line':
                     ims.append(
                         axes[i].plot(
-                            v[1].cpu().numpy()[n_neurons[v[0]][0]:n_neurons[v[0]][1], time[0]:time[1]].cpu().numpy().T
+                            v[1].cpu().numpy()[n_neurons[v[0]][0]:n_neurons[v[0]][1], time[0]:time[1]].T

```

Comment:
This won't work when `spikes` maps from `str`s to `torch.Tensor`s. A better idea would be to convert all input data to `numpy` arrays at the beginning of the function, if they're not already.

Reply:
Not sure I understand. `v1[1]` is `torch.tensor` that gets converted to `numpy` array with `v[1].cpu().numpy()`. Then `v[1].cpu().numpy()[...]` is already a `numpy` array.

Converting `voltages` values to `numpy` at the start will probably be more efficient.

Reply:
Oh I see, that makes sense.